src/utils/helpers/index.ts

```diff
@@ -34,6 +34,11 @@ export const sanitizeNumberInput = (
   else return sanitized;
 };
 
+export const sanitizeInputOnBlur = (value: string) => {
```

Comment:
1. Why not use `sanitizeNumberInput` (and also handle "." there)?
2. I'd rename this function. It's in utils should it shouldn't "dictate" when it's being used.

Reply:
1. `sanitizeNumberInput` is used on change. "." should only be transformed to "0" only after the user focus out, else user won't be able to write `.23`.
`Number(".") === NaN` so I need a specific check for it instead of always do `Number(value).toString()`.
2. The function is (and should be) tightly link with the input cycle check (so is `sanitizeNumberInput`). Now that you mention it, I think the best would be to rename `sanitizeNumberInput` to `sanitizeNumberInputOnChange`. I can move these functions in a dedicated "input-number.ts" file.

Reply:
Could only be an object as they are supposed to be used in combination : 
```typescript
export const sanitizeNumberInput = {
  onChange: () => ,
  onBlur: () => ,
}
```

Reply:
sounds like a plan